.github/label-pr.yml

```diff
@@ -0,0 +1,12 @@
+- regExp: ".*\\.sql+$"
+  labels: ["DB"]
+- regExp: "^(src\\server\\scripts\\.(cpp|h))$"
```

Comment:
@FrancescoBorzi

Reply:
@Kitzunu is this supposed to exclude the scripts? where is the part of "including all cpp/h files" ?

Reply:
I just thought so too. Will look mord into it when Im not trying to fix it on my phone xd